shishito/runtime/shishito_support.py

```diff
@@ -108,16 +120,14 @@ def get_environment_config(self):
         config.read(config_path)
         return config
 
-    def get_modules(self, platform=None, environment=None, module=None):
-        if platform is None:
-            platform = self.test_platform
-
-        if environment is None:
-            environment = self.test_environment
+    def get_modules(self, module):
```

Comment:
get_modules is always called with arg which returns one module 
(so fallback branch returning module dict is never used - remove it)

it would be also better name it get_module() if it's return just one module